metasyn/metaframe.py

```diff
@@ -1,4 +1,4 @@
-"""Module defining the MetaFrame class, used for the conversion of DataFrames to MetaFrames."""
+"""Conversion of DataFrames to MetaFrames."""  # pylint: disable=invalid-name
```

Comment:
Can you explain a bit why you changed the docstring? To me it seems like there is more in this module then conversion of dataframes.

Reply:
I think I was actually sloppy here, and forgot that I was still working on this part and never looked back to it. Updated it again.